service/controller/resource/tccpazs/create.go

```diff
@@ -137,6 +137,11 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		}
 	}
 
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("CP azs %#v, azs from subnets %#v", key.ControlPlaneAvailabilityZones(cp), azsFromSubnets(cc.Status.TenantCluster.TCCP.Subnets)))
+	if len(mds) > 0 {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("mds azs %#v", key.MachineDeploymentAvailabilityZones(mds[0])))
+	}
```

Comment:
Do we need that? In case we want to log more information we need to find a structure that makes sense. Otherwise we have to deal with some hard to read mess when shit hits the fan and that could just be better.